kube/src/config/incluster_config.rs

```diff
@@ -32,17 +33,11 @@ pub fn load_token() -> Result<String> {
 }
 
 /// Returns certification from specified path in cluster.
-pub fn load_cert() -> Result<Vec<Certificate>> {
+pub fn load_cert() -> Result<Vec<Der>> {
```

Comment:
ah, this was a reqwest::Certificate before, guess that wouldn't work with tungstenite.
i've only had a quick look at the cert juggling so far, but it makes sense why you have had to make these changes at least on the surface level.

Reply:
Yeah, it's now converted when creating the client. This allows TLS Connector to use the data as well.